tests/drivers/i2s/i2s_speed/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,74 @@
+/* i2s-node0 is the receiver    */
+/* i2s-node1 is the transmitter */
+/ {
+	aliases {
+		i2s-node0 = &i2s1;
+		i2s-node1 = &i2s2;
+	};
+};
+
+&pll1 {
+	div-m = <5>;
+	mul-n = <64>;
+	div-p = <2>;
+	div-q = <4>;
+	div-r = <2>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+&i2s1 {
```

Comment:
Why are `interrupts` and `dma-names` defined again in the nodes?

Reply:
- DMA characteristics ( (size of source (8 , 16 or 32 bits) size of destination (8 , 16 or 32 bits), the source and the destination shall be incremented or not after each DMA transfer...) depend on the application or feature (I2S, Spi, Uart...)
- - the interrupts is defined  again in the node because the priority level is set according to the involved interrupts : I2s interrupts, DMA RX and Tx interrupts....